taxi/views.py

```diff
@@ -34,12 +35,52 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `CreateView` and `UpdateView` exposes all fields of the model in the form. Ensure that this is intended and that no sensitive fields are included. Consider specifying only the fields that should be editable by the user.